editorconfig-core.el

```diff
@@ -80,24 +80,20 @@ RESULT is used internally and normally should not be used."
   (let ((handle (editorconfig-core-handle (concat (file-name-as-directory dir)
                                                   confname)))
         (parent (file-name-directory (directory-file-name dir))))
-    (if (or (string= parent
-                     dir)
-            (and handle
-                 (editorconfig-core-handle-root-p handle)))
-        (cl-remove-if-not 'identity
-                          (cons handle result))
+    (if (or (string= parent dir)
```

Comment:
`when`?

Reply:
We can't use `when` here, there are two conditions with that operation. `if` is the right operation to use.

Reply:
We can't use `when` here, there are two conditions with that operation. `if` is the right operation to use.